core/src/epicli/data/common/ansible/playbooks/roles/preflight/tasks/main.yml

```diff
@@ -1,6 +1,32 @@
 ---
 - include_tasks: check-user.yml
 
+- name: Set is architecture supported to true
+  set_fact:
+     is_arch_supported: true
+  changed_when: true
```

Comment:
This parameter has no sense for `set_fact` task since it doesn't change state.

Reply:
it actually changes when found components not listed